spec/user_stories_spec.rb

```diff
@@ -0,0 +1,82 @@
+# As a Maker
+# So that I can post messages on Chitter as me
+# I want to sign up for Chitter
+feature 'Sign up' do
+  scenario 'display welcome message to new users' do
+    sign_up
+    expect(page).to have_content('Welcome chuck')
+  end
+
+  scenario 'increase user count by one upon sign up' do
+    expect{ sign_up }.to change(User, :count).by(1)
+  end
+
+  scenario 'password does not match password confirmation' do
+      expect { sign_up(password_confirmation: 'wrong') }.not_to change(User, :count)
+      expect(current_path).to eq('/sign_up') # current_path is a helper provided by Capybara
+      expect(page).to have_content 'Password does not match the confirmation'
+    end
```

Comment:
Lint/BlockAlignment: `end` at 18, 4 is not aligned with `scenario 'password does not match password confirmation' do` at 14, 2.